src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisParametersService.java

```diff
@@ -0,0 +1,147 @@
+        securityAnalysisParameters.setIncreasedViolationsParameters(getIncreasedViolationsParameters(entity.getFlowProportionalThreshold(), entity.getLowVoltageProportionalThreshold(), entity.getLowVoltageAbsoluteThreshold(), entity.getHighVoltageProportionalThreshold(), entity.getHighVoltageAbsoluteThreshold()));
+        return securityAnalysisParameters;
+    }
+
+    public static SecurityAnalysisParameters.IncreasedViolationsParameters getIncreasedViolationsParameters(double flowProportionalThreshold, double lowVoltageProportionalThreshold, double lowVoltageAbsoluteThreshold, double highVoltageProportionalThreshold, double highVoltageAbsoluteThreshold) {
+        SecurityAnalysisParameters.IncreasedViolationsParameters increasedViolationsParameters = new SecurityAnalysisParameters.IncreasedViolationsParameters();
+        increasedViolationsParameters.setFlowProportionalThreshold(flowProportionalThreshold);
+        increasedViolationsParameters.setLowVoltageAbsoluteThreshold(lowVoltageAbsoluteThreshold);
+        increasedViolationsParameters.setLowVoltageProportionalThreshold(lowVoltageProportionalThreshold);
+        increasedViolationsParameters.setHighVoltageAbsoluteThreshold(highVoltageAbsoluteThreshold);
+        increasedViolationsParameters.setHighVoltageProportionalThreshold(highVoltageProportionalThreshold);
+        return increasedViolationsParameters;
+    }
+
+    public static SecurityAnalysisParametersValues fromEntity(SecurityAnalysisParametersEntity entity) {
+        Objects.requireNonNull(entity);
+        return SecurityAnalysisParametersValues.builder()
+                .lowVoltageAbsoluteThreshold(entity.getLowVoltageAbsoluteThreshold())
+                .lowVoltageProportionalThreshold(entity.getLowVoltageProportionalThreshold())
+                .highVoltageAbsoluteThreshold(entity.getHighVoltageAbsoluteThreshold())
+                .highVoltageProportionalThreshold(entity.getHighVoltageProportionalThreshold())
+                .flowProportionalThreshold(entity.getFlowProportionalThreshold())
+                .build();
+    }
+
+    public static SecurityAnalysisParametersValues getDefaultSecurityAnalysisParametersValues() {
+        return SecurityAnalysisParametersValues.builder()
+                .lowVoltageAbsoluteThreshold(DEFAULT_LOW_VOLTAGE_ABSOLUTE_THRESHOLD)
+                .lowVoltageProportionalThreshold(DEFAULT_LOW_VOLTAGE_PROPORTIONAL_THRESHOLD)
+                .highVoltageAbsoluteThreshold(DEFAULT_HIGH_VOLTAGE_ABSOLUTE_THRESHOLD)
+                .highVoltageProportionalThreshold(DEFAULT_HIGH_VOLTAGE_PROPORTIONAL_THRESHOLD)
+                .flowProportionalThreshold(DEFAULT_FLOW_PROPORTIONAL_THRESHOLD)
+                .build();
+    }
+
+    public SecurityAnalysisParametersValues getParameters(UUID parametersUuid) {
+        var params = securityAnalysisParametersRepository.findById(parametersUuid).orElseThrow(() -> new SecurityAnalysisException(PARAMETERS_NOT_FOUND));
```

Comment:
Why not managing the Optional in the controller and just sending back 404 ?
I think we should all do the same (it's done like that in voltage-init and I did the same)
https://github.com/gridsuite/voltage-init-server/pull/13/files#diff-df82c2e418575c219f0fe0ec260f471c33303cda9064ef8a7cadb58b9a486769R57
https://github.com/gridsuite/sensitivity-analysis-server/blob/9107dc2c7ae207e4858136f6ddc330730f1f8071/src/main/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisParametersController.java#L67-L69